db/package_detail.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Think you could add "dummy" in the file name ? to make sure it doesn't get mixed in with seed data :)

or a dummy/ folder works too

Reply:
sure @patrixr

Reply:
Thanks :)